Makefile

```diff
@@ -93,8 +92,7 @@ run: server
 	./$(APP_NAME)
 
 test: vendor glide.lock
-	go test $(GOFILES)	# TODO: move this into a sub-test inside test/*.sh
-	./test.sh
+	./test.sh $(TESTOPTIONS)
```

Comment:
Is $TESTOPTIONS missing?

Reply:
I think it can be set from the caller, and it can be interpreted by test scripts? Not implemented yet?

Reply:
It is meant to be specified on the command line, for example: make test TESTOPTIONS=-vx . It is a style I've used in the past. Do you think adding a comment to that effect here would be helpful?